Misc/NEWS.d/next/Library/2018-10-29-04-14-58.bpo-35101.xImh8d.rst

```diff
@@ -0,0 +1,2 @@
+Fix inspect.findsource incorrectly returning the line number of the first
+function above any given class frame objects.
```

Comment:
`inspect.findsource` remains undocumented now. I'd suggest simply saying that the patch fixes finding class object frames in the inspect module. 
```suggestion
Fix bug where the :mod:`inspect` module failed to identify the
correct source code for class object frames.
```